python/tank/platform/engine.py

```diff
@@ -970,6 +970,8 @@ def register_command(self, name, callback, properties=None):
                 # we know the app for the existing item.
                 # so prefix with app name
                 prefix = existing_item["properties"].get("app").instance_name
+                if existing_item["properties"].get("group"):
+                    prefix = "%s:%s" % (prefix, existing_item["properties"]["group"])
```

Comment:
Cool! Looked at the code and this is nice - it works because desktop is using the `title` property to format things. If you look at the doc string, `title` is under the section of things that are supposedly only used by the shotgun engine. Please move it up to the general section to reflect the fact that it's used in more than one place.